### ISSUE TITLE
Not working with autocompete-plus

### ISSUE THREAD

**Issue by Shmasya** [emmetio/emmet-atom#409]
When I see popup with autocomplete-plus I press Enter or Tab and I proishodt something strange. Emmet goes on a new line or does my <div class="row"></div> something <div class="<row>"></div>. Who can tell that?

**Comment by kumarharsh** [comment]
Yes, emmet is very frustrating in atom. While editing CSS, autocomplete shows some suggestions, and if you press <Tab> something entirely different is inserted by emmet.

**Comment by MartinMuzatko** [comment]
Reported it here first: https://github.com/atom/autocomplete-css/issues/45#issuecomment-207179225
Thought it is autocompletes fault.

**Comment by sergeche** [comment]
Not exactly. Atom and autocomplete-plus devs did a great job on autocompletion since initial Emmet release. I have to re-write Emmet plugin to better integrate with autocomplete-plus plugin.

**Comment by MartinMuzatko** [comment]
I wonder how tab is supposed to work in CSS? I always used emmet for html.

**Comment by sergeche** [comment]
Emmet is able to detect context for tab handler (e.g. whether you’re expanding in context of property name or property value) so it should expand in property context only

**Comment by hugollm** [comment]
Can't we just have an option to disable CSS abbreviations? At least for now...

**Comment by MartinMuzatko** [comment]
This issue has spawned a bunch of "emmet lite" plugins. Such as https://atom.io/packages/emmet-simplified

I'd also agree with having an option to turn different features on/off. Keybinding management is a hell of a mess at the moment anyway. Enabling emmet in more than html is also hell at the moment. So, this would be a nice alternative to dealing with config files for hours.

**Comment by sergeche** [comment]
I have plans for complete re-write of Emmet and its plugins in near future to some the very same issues
